README.md

```diff
@@ -10,7 +10,12 @@ The simplest case is to specify user name and password when starting
 the container:
 
 ```
-docker run -p 11021:21 -it --rm -e FTP_USER=scott -e FTP_PASS=tiger -e HOST=localhost mcreations/openwrt-ftp
+docker run -it --rm \
+            -p 11021:21 \
+            -p 65100:65100 -p 65101:65101 -p 65102:65102 -p 65103:65103 -p 65104:65104 \
```

Comment:
The ports which are published start at 65100, but the value of PASV_MIN_PORT is 65000.

Is this a typo or do I misunderstand?